gulpfile.js

```diff
@@ -42,4 +44,16 @@ gulp.task('serve', function() {
     }));
 });
 
+gulp.task('sass', function() {
+  gulp.src(['./sass/*.sass', './sass/*.scss'])
+      .on('error', function(e) {
+        handleError(e, this);
+      })
+      .pipe(sourcemaps.init({loadMaps: true}))
+      .pipe(sass({indentationSyntax: true}))
+      .pipe(autoprefixer('last 2 versions'))
+      .pipe(sourcemaps.write('./dist'))
+      .pipe(gulp.dest('./css/'))
```

Comment:
I'm sorry. I probably wasn't clear enough. I meant that the ```gulp.dest``` would point to the ```./dist``` folder. And the sourcemaps would write to ```./``` the same way it's done [here](https://github.com/pokemon-club/admin/pull/8/files#diff-b9e12334e9eafd8341a6107dd98510c9L29)